core-bundle/src/DataContainer/DataContainerOperation.php

```diff
@@ -28,7 +28,7 @@ class DataContainerOperation implements \ArrayAccess
      */
     public function __construct(private readonly string $name, array $operation, private readonly array $record, private readonly DataContainer $dataContainer)
     {
-        $id = StringUtil::specialchars(rawurldecode((string) $record['id']));
+        $id = StringUtil::specialchars(rawurldecode((string) ($record['id'] ?? '')));
```

Comment:
So if I read this correctly, `$record['id']` can now be not set, whereas it was always present before. Shouldn‘t we handle this case explicitly? Because the `$id` variable is used several times in the method.

Reply:
I only added this because I got a warning in the tests, because tests don't set that data. I think it is always present in real code.